### DISCUSSION TITLE
Autoinference from docs

### DISCUSSION THREAD

**Discussion by matthieu-perso** [APIs-guru/openapi-directory#1126] [General]
Hey team, 
How did you retro engineer the specs from the API ? Do you have an opensource package for this ? 
Matthieu

**Comment by MikeRalphson** [comment]
95% of the OpenAPI documents are official ones released by the API providers. A few are third-party definitions generally created by hand. There is an open-source package for converting HTML documentation into OpenAPI - https://github.com/postman-open-technologies/scrape-to-oas
